### ISSUE TITLE
nginx-module-brotli depends on older nginx on Ubuntu 18.04

### ISSUE THREAD

**Issue by silverwind** [cryptofuture/nginx-hda-bundle#26]
On Ubuntu 18.04, this PPA installs nginx 1.14.0 but `nginx-module-brotli` seems to depend on a older unavailable version of nginx, preventing the module from being installed. Is this intentional?

````
$ apt install nginx-module-brotli
Reading package lists... Done
Building dependency tree
Reading state information... Done
Some packages could not be installed. This may mean that you have
requested an impossible situation or if you are using the unstable
distribution that some required packages have not yet been created
or been moved out of Incoming.
The following information may help to resolve the situation:

The following packages have unmet dependencies:
 nginx-module-brotli : Depends: nginx (= 1.13.8-2-ppa7~bionic)
E: Unable to correct problems, you have held broken packages.

````

**Comment by cryptofuture** [comment]
Weird, I hope update would be finally available tomorrow. And it would be nice if you would test 18.04. Unfortunately, I decided not to upgrade to 18.04, due too small (60Gb) ssd and other installations on 16.04.x still. Finished with xenial build locally, but bionic requires even more patches on top of modules again. 
Gladly I was able to free around 2 Gb, and mount  `/var/cache/pbuilder` in the tmpfs
